lib/ash_admin/resource/resource.ex

```diff
@@ -173,7 +174,7 @@ defmodule AshAdmin.Resource do
   end
 
   def relationship_select_max_items(resource) do
-    Spark.Dsl.Extension.get_opt(resource, [:admin], :relationship_select_max_items, 50, true)
+    Spark.Dsl.Extension.get_opt(resource, [:admin], :relationship_select_max_items, nil, true)
```

Comment:
```suggestion
    Spark.Dsl.Extension.get_opt(resource, [:admin], :relationship_select_max_items, 50, true)
```

We do actually want to repeat this here. In newer DSLs these functions aren't hand-written so they don't have this problem, but in older DSLs w/ hand-written info modules, this info is duplicated.